.github/workflows/nightly.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Nightly Releases'
+
+on:
+  schedule:
+    - cron: '0 0 * * 2-6'
```

Comment:
This should be 1-5 not 2-6 as this would make it Tuesday through Saturday.

https://crontab.guru/#0_0_*_*_1-5
https://crontab.guru/#0_0_*_*_2-6

Reply:
Yes since it's at 0 0 we want it to run from tuesday to Saturday (Friday 23h59 + 1min) :)

Reply:
We don't want it on Monday morning?

Reply:
We don't merge anything on Sunday so it would be useless :)